cmd/main_test.go

```diff
@@ -34,20 +35,20 @@ import (
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 )
 
-type HTTPResponse struct {
-	Body       string
-	StatusCode int
+type NamespaceRoleBinding struct {
+	Namespace   string
+	Role        string
+	RoleBinding string
 }
 
 type HTTPheader struct {
 	name  string
 	value string
 }
 
-type NamespaceRoleBinding struct {
-	Namespace   string
-	Role        string
-	RoleBinding string
+type HTTPResponse struct {
```

Comment:
Any reason we changed the order here?

Reply:
No, I first placed these in different file and again moved here. So, the order changed will change it to the original order.